### DISCUSSION TITLE
WiFi strength WT32-SC01 Plus ?

### DISCUSSION THREAD

**Discussion by stovedoctor** [HASwitchPlate/openHASP#475] [Hardware]
Is it just me ?
Setting up WT32-SC01 Plus at my PC that is 1 wall about 10 feet away from my Google nest router all is good.
Move it to same open room but 25 feet away signal bad and keep disconnecting. I don't even have a Inclosure over it yet ...![Screenshot_20230425-181552.png](https://user-images.githubusercontent.com/60519485/234441284-82ad1cb3-511b-4243-89c7-cd03369f48a4.png)

![Screenshot_20230425-181220.png](https://user-images.githubusercontent.com/60519485/234441307-3deed817-70cd-41ec-b45b-1fcf15761e5c.png)

 WTF ?

**Comment by stovedoctor** [comment]
I have no WiFi issue with 55 smart devices in my 1900 sw foot wood framed ranch house. Router in center. 
Ranging from 6 years old 8266 node mcu, D1 minus, 
Esp32s and Sonoff switches. Some over 75 feet away from my router.

**Comment by stovedoctor** [comment]
![PXL_20230426_012259707~2.jpg](https://user-images.githubusercontent.com/60519485/234442477-01414b5b-7203-41ec-b7bf-28e6e33801d3.jpg)

![PXL_20230426_012203678.jpg](https://user-images.githubusercontent.com/60519485/234442493-38d83152-53de-4720-a3bd-4d56e044eacb.jpg)

**Comment by nagyrobi** [comment]
You might also have _interference_.

**Reply by stovedoctor** [reply]
Maybe but my desk in same open room has so much more electronics around it. 

I'm going to set up another one and run some test.
As soon as I can figure out some code to run a graph on HA of WiFi strength in each device.

**Comment by stovedoctor** [comment]
So I set up another unit 
Placed it in office.
Good signal
This first one I mounted on a wall 6 feet away from router and shitty signal.
Must be a bad antenna ![Screenshot_20230426-214309.png](https://user-images.githubusercontent.com/60519485/234761986-c0cdd879-1621-4dc2-bf86-8e2caa128bc2.png)

![PXL_20230426_230315220.jpg](https://user-images.githubusercontent.com/60519485/234762012-2acde4c6-c283-4fde-b092-be0d30cb1b18.jpg)

![Screenshot_20230426-185004.png](https://user-images.githubusercontent.com/60519485/234762025-8ab1728f-f69d-4ae3-a8e5-ac1f5e2e6d55.png)

**Comment by nagyrobi** [comment]
Antennas have characteristics - to set the direction which signal is stronger. Moreover, if your AP supports MIMO with multiple antennas, it can direct the "beam" towards your devices, which can turn out badly if you have stuff on the other side too.

**Reply by stovedoctor** [reply]
I have 55 other esp32 or 8266 devices with built in antennas from old node mcu to d1mini around the house same WiFi up to 75 feet away no issues.

**Reply by nagyrobi** [reply]
Yeah... Maybe this unit is the culprit

**Comment by stovedoctor** [comment]
It's working ok -72dBm
Power supply is 2.5 amp

![PXL_20230427_165522078~2.jpg](https://user-images.githubusercontent.com/60519485/234936165-661aea35-ad0e-4fbf-8548-ad56f1a08add.jpg)

 I guess  this one stays here 😂

**Reply by stovedoctor** [reply]
anyone know how to fix the Antenna or what ever is wrong on this ONE unit? my other one works great this one has bad signal.
